rotkehlchen/chain/evm/decoding/pendle/decoder.py

```diff
@@ -390,11 +416,109 @@ def _decode_exit_post_exp_to_token(self, context: DecoderContext) -> DecodingOut
             output_token_log_idx=2,
         )
 
+    def _decode_claim_rewards(self, context: DecoderContext) -> DecodingOutput:
+        if context.tx_log.topics[0] != REDEEM_REWARDS_TOPIC:
+            return DEFAULT_DECODING_OUTPUT
+
+        raw_rewards_amount = int.from_bytes(context.tx_log.data[64:])
+        for event in context.decoded_events:
+            if (
+                    event.address in self.pools and
+                    event.event_type == HistoryEventType.RECEIVE and
+                    event.event_subtype == HistoryEventSubType.NONE and
+                    asset_raw_value(
+                        asset=(crypto_asset := event.asset.resolve_to_crypto_asset()),
+                        amount=event.amount,
```

Comment:
do you need here `get_token_or_native`?

Reply:
where? i don't know the asset that's been claimed so i have to convert to the raw amount.